examples/demo.js

```diff
@@ -6,16 +6,16 @@ var defaultHtmlSequencerUi = require('./lib/defaultHtmlSequencerUi.js'),
   insertPreview = require('./lib/insertPreview.js');
 
 window.onload = function () {
-  sequencer = ImageSequencer();
+  sequencer = ImageSequencer(); // Set the global sequencer variable.
```

Comment:
You forget to remove full stop in phrases in this PR too

Reply:
Set is a verb so shouldn't it be considered as a full sentence?

Reply:
I think it is a phrase as it doesn't have subject before predicate(sentence = subject + predicate)

Reply:
```suggestion
  sequencer = ImageSequencer(); // Set the global sequencer variable
```